### PR TITLE
Move Activity definitions to constants

### PR DESCRIPTION
**Pull request by nupplaphil**
I moved the definition of the Activity Verbs, Object Types and Namespaces to different classes.

I tried to research the whole definition and specifications about the different protocols and add them as reference in the PHP DOC.

It's a FollowUp based on the suggestion of https://github.com/friendica/friendica/pull/7765#discussion_r337820358

I tripple checked the occurences of all definitions through the codebase, hopefully I got all of them :-)

### CODE REVIEW

**Comment by nupplaphil** [comment]
Needs https://github.com/friendica/friendica-addons/pull/905 too

File: src/Protocol/Activity/ANamespace.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Friendica\Protocol\Activity;
+
+/**
+ * Activity namespaces constants
+ */
+final class ANamespace
```
Comment (MrPetovan) [inline src/Protocol/Activity/ANamespace.php]: AaaaaaaaaaaaaaaaAAAaaaaaAaaaaah no.

Please move this to `Friendica\Protocol\ActivityNamespace`

File: src/Protocol/Activity/ActivityNamespace.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 /**
  * Activity namespaces constants
  */
-final class ANamespace
+final class ActivityNamespace
```
Comment (MrPetovan) [inline src/Protocol/Activity/ActivityNamespace.php]: Why not just `Friendica\Protocol\ActivityNamespace`? is there anything else in the `Friendica\Protocol\Activity` namespace?

Comment (nupplaphil) [reply src/Protocol/Activity/ActivityNamespace.php]: Yes, the `ObjectType` class

Comment (nupplaphil) [reply src/Protocol/Activity/ActivityNamespace.php]: tbh I thought @annando doesn't like such long namespaces... At first I had `Friendica\Protocol\Activity\NS`.. but this shortname must not be used ...

**Pull request merged**